webapp/packages/core-authentication/src/PasswordPolicyService.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { computed, makeObservable } from 'mobx';
+
+import { injectable } from '@cloudbeaver/core-di';
+import { LocalizationService } from '@cloudbeaver/core-localization';
+import { ServerConfigResource } from '@cloudbeaver/core-root';
+import type { PasswordPolicyConfig } from '@cloudbeaver/core-sdk';
+
+const DEFAULT_PASSWORD_POLICY: PasswordPolicyConfig = {
+  minLength: 8,
+  requiresUpperLowerCase: false,
+  minDigits: 0,
+  minSpecialCharacters: 0,
+};
+
+type ValidationResult = { isValid: true; errorMessage: null } | { isValid: false; errorMessage: string };
```

Comment:
why not to use:
```ts
interface IPasswordValidationSatus {
  isValid: boolean;
  message: string;
}
```

Reply:
Because there is no message if the status is valid
`if(isValid) {no message hee}`